pkgs/by-name/jf/jfrog-cli/package.nix

```
@@ -7,17 +7,17 @@
 
 buildGo123Module rec {
```

```suggestion
buildGoModule rec {
```
buildGoModule = buildGo123Module now.